runtime/turing/src/xcm_config.rs

```diff
@@ -220,7 +222,7 @@ parameter_types! {
 	pub KarPerSecond: (AssetId, u128) = (
 		MultiLocation::new(
 			1,
-			X2(Parachain(parachains::karura::ID), GeneralKey(parachains::karura::KAR_KEY.to_vec())),
+			X2(Parachain(parachains::karura::ID), GeneralKey(WeakBoundedVec::<u8, ConstU32<32>>::force_from(parachains::karura::KAR_KEY.to_vec(), None))),
```

Comment:
they changed `GeneralKey` to take a `WeakBoundedVec` instead of a `Vec`

Reply:
`GeneralKey` now takes a `WeakBoundedVec` with a "max" length of 32. I put max in quotes as the `force_from` will log the issue but do it anyway